onebusaway-android/src/main/java/org/onebusaway/android/ui/TripPlanFragment.java

```diff
@@ -127,6 +136,45 @@ interface Listener {
 
     private FirebaseAnalytics mFirebaseAnalytics;
 
+    //sets the address in the text view from the user selected contact
+    private final String ID = "id";
+    private ActivityResultLauncher<TextView> mSelectAddressFromContactLauncher = registerForActivityResult(
+            new ActivityResultContract<TextView, Intent>() {
+                int tvId;
```

Comment:
Just a quick thought when it comes to naming, think about how it might feel to read this code a couple of years down the line. Would `tvId` instantly make sense? Something like `addressInputId` could make it much clearer and easier to follow.